src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt

```diff
@@ -149,6 +146,14 @@ class Stylesheet(private val stylesheetPath: String?, private val symbolTableVis
     fun getDisplayNewLinesInCode(): Boolean = stylesheet.displayNewLinesInCode
 
     fun getTabSpacing(): Int = stylesheet.tabSpacing
+
+    fun renderDataStructure(identifier: String): Boolean {
+        return if (stylesheet.positions.containsKey(identifier)) {
```

Comment:
rest of logic looks fine could be worth replacing this with && though

Reply:
return !stylesheet.positions.containsKey(identifier) || stylesheet.positions[identifier]!!.height != 0.0

Reply:
nitpick up to you to incorporate or not